src/source/feeds.rst

```diff
@@ -31,6 +78,635 @@ XML file.
+======= =============================
+
+.. index:: minEnergyEfficiencyClass
+.. _feed_minEnergyEfficiencyClass: 
+
+minEnergyEfficiencyClass
+""""""""""""""""""""""""
+
+Minimal energy efficiency class. Possible values defined in :ref:`feed_energyEfficiencyClass`
+
+======= =============================
+Example <ad:minEnergyEfficiencyClass>E</ad:minEnergyEfficiencyClass>
+======= =============================
+
+.. index:: maxEnergyEfficiencyClass
+.. _feed_maxEnergyEfficiencyClass: 
+
+maxEnergyEfficiencyClass
+""""""""""""""""""""""""
+
+Maximal energy efficiency class. Possible values defined in :ref:`feed_energyEfficiencyClass`
+
+======= =============================
+Example <ad:maxEnergyEfficiencyClass>B</ad:maxEnergyEfficiencyClass>
+======= =============================
+
+.. index:: color
+.. _feed_color: 
+
+color
+""""""""""""""""""""""""
+
+Color definition follows `Google Merchant Center <https://support.google.com/merchants/answer/7052112>`_ guidelines.
+String identifier max 1-100 chars.
+
+======= =============================
+Example <ad:color>Black</ad:color>
+======= =============================
+
+.. index:: gender
+.. _feed_gender: 
+
+gender
+""""""""""""""""""""""""
+
+Gender definition follows `Google Merchant Center <https://support.google.com/merchants/answer/7052112>`_ guidelines.
+
+Allowed values: *male*, *female*, *unisex*
+
+======= =============================
+Example <ad:gender>unisex</ad:gender>
+======= =============================
+
+.. index:: ageGroup
+.. _feed_ageGroup: 
+
+ageGroup
+""""""""""""""""""""""""
+
+Age group definition follows `Google Merchant Center <https://support.google.com/merchants/answer/7052112>`_ guidelines.
+
+Allowed values: *newborn*, *infant*, *toddler*, *children*, *adult*
+
+======= =============================
+Example <ad:ageGroup>adult</ad:ageGroup>
+======= =============================
+
+.. index:: size
+.. _feed_size: 
+
+size
+""""""""""""""""""""""""
+
+Size definition follows `Google Merchant Center <https://support.google.com/merchants/answer/7052112>`_ guidelines.
+String identifier max 1-100 chars.
+
+.. index:: unitPricingBaseMeasure
+.. _feed_unitPricingBaseMeasure: 
+
+======= =============================
+Example <ad:size>S</ad:size>
+======= =============================
+
+unitPricingBaseMeasure
+""""""""""""""""""""""""
+
+The denominator for product unit price. See `Google Merchant Center <https://support.google.com/merchants/answer/7052112>`_
+
+======= =============================
+Example <ad:unitPricingBaseMeasure>1kg</ad:unitPricingBaseMeasure>
+======= =============================
+
+.. index:: unitPricingMeasure
+.. _feed_unitPricingMeasure:         
+
+unitPricingMeasure
+""""""""""""""""""""""""
+
+Defines the measure and dimension of the product. Example 125ml, 100g. See `Google Merchant Center <https://support.google.com/merchants/answer/7052112>`_
+
+======= =============================
```

Comment:
just wondering, why don't you use .. code-block: xml for these examples?

Reply:
Css for the tables in combination with one line code blocks does not look impressive, but it makes sense to make it look uniform. I am changing that.